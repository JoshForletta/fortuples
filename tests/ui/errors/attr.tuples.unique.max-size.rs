use fortuples::fortuples;

struct Container<T>(T);

fortuples! {
    #[tuples::max_size(1)]
    #[tuples::max_size(2)]
    impl Container<#Tuple> {}
}

fn main() {}
