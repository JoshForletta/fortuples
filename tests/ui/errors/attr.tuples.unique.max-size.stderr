error: `max_size` should be set only once
 --> tests/ui/errors/attr.tuples.unique.max-size.rs:7:15
  |
7 |     #[tuples::max_size(2)]
  |               ^^^^^^^^

error: the first definition was here
 --> tests/ui/errors/attr.tuples.unique.max-size.rs:6:15
  |
6 |     #[tuples::max_size(1)]
  |               ^^^^^^^^
